Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/Installer.java

```diff
@@ -59,21 +60,38 @@ private Installer() {
     @NbBundle.Messages({
         "Installer.initialCreateSqlite.title=Enable Central Repository?",
         "Installer.initialCreateSqlite.messageHeader=The Central Repository is not enabled. Would you like to enable it?",
-        "Installer.initialCreateSqlite.messageDesc=It will store information about all hashes and identifiers that you process. " +
-            "You can use this to ignore previously seen files and make connections between cases."
+        "Installer.initialCreateSqlite.messageDesc=It will store information about all hashes and identifiers that you process. "
+        + "You can use this to ignore previously seen files and make connections between cases."
     })
     @Override
     public void restored() {
         Case.addPropertyChangeListener(pcl);
         ieListener.installListeners();
+        if (Version.getBuildType() == Version.Type.RELEASE) {
+            centralRepoCheckAndSetup();
+        }
+        // now run regular module startup code
+        try {
+            CentralRepoDbManager.upgradeDatabase();
+        } catch (CentralRepoException ex) {
+            LOGGER.log(Level.SEVERE, "There was an error while upgrading the central repository database", ex);
+            if (RuntimeProperties.runningWithGUI()) {
```

Comment:
I  notice that this call to reportUpgradeError is conditional on RuntimeProperties.runningWithGUI, which it should be. However, the call to reportUpgradeError on line 143 is not conditional in this way. Maybe it would be better to put a  RuntimeProperties.runningWithGUI conditional in reportUpgradeError.

Reply:
I'm going to accept this PR and put in the requested change (fix) in a separate PR for the sake of this weekend's automated testing.